README.md

```diff
@@ -47,9 +47,15 @@ resolvers += Resolver.bintrayRepo("otherUser", "maven")
 
 ## Usage
 
+Note that when specifying `sbt-bintray` settings in your build definition, you will need to add the following import:
```

Comment:
Could you change this to specify that this is only required in `project/*.scala` files, please?

It isn't required in build.sbt because it's auto-imported: https://github.com/sbt/sbt-bintray/blob/v0.4.0/src/main/scala/BintrayPlugin.scala#L21